src/Features/CSharp/Portable/EditAndContinue/SyntaxUtilities.cs

```diff
@@ -187,7 +187,13 @@ public static SyntaxNode TryGetEffectiveGetterBody(ArrowExpressionClauseSyntax p
                 return propertyBody.Expression;
             }
 
-            return accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault()?.Body;
+            var firstGetter = accessorList?.Accessors.Where(a => a.IsKind(SyntaxKind.GetAccessorDeclaration)).FirstOrDefault();
+            if (firstGetter == null)
+            {
+                return null;
+            }
+
+            return (SyntaxNode)firstGetter.Body ?? firstGetter.ExpressionBody?.Expression;
```

Comment:
It looks like we're checking in a few places for a non-`null` `ExpressionBody` with a `null` expression. Is this actually possible? If so, I think there should be a test for this.